documentation/modules/exploit/windows/http/manageengine_adaudit_plus_authenticated_rce.md

```diff
@@ -0,0 +1,134 @@
+## Vulnerable Application
+The module exploits security issues in ManageEngine ADAudit Plus prior to 7006 that allow authenticated users to execute arbitrary code
+by creating a custom alert profile and leveraging the custom alert script component.
+
+The module first runs a few checks to test the provided credentials, retrieve the configured domain(s) and obtain the build number.
+If the credentials are valid and the target is vulnerable, the module creates an alert profile that will be triggered
+for any failed login attempt to the configured domain.
+
+For versions prior to build 7004, the payload is directly inserted in the custom alert script component of the alert profile.
+
+For builds 7004 and 7005, the module leverages an arbitrary file write vulnerability (CVE-2021-42847) to create a Powershell script
+in the alert_scripts directory that contains the payload.
+The name of this script is then provided as the value for the custom alert script component of the alert profile.
+
+The module will automatically delete the created alert profile before completing. This happens even if no shell was obtained.
+
+This module requires valid credentials for an account with the privileges to create alert scripts.
+It has been successfully tested against ManageEngine ADAudit Plus builds 7003 and 7005 running on Windows Server 2012 R2.
+
+## Installation Information
```

Comment:
Thought this was meant to be a subsection of vulnerable application section? Could be wrong though.